code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm

```diff
@@ -1660,6 +1664,19 @@ var/list/cult_spires = list()
 		if (3)
 			takeDamage(10)
 
+/obj/structure/cult/bloodstone/singularity_act(var/singularity_size=0,var/obj/machinery/singularity/S)
+	switch(singularity_size)
+		if(1 to 4)
+			ex_act(3)
+		if(5 to 8)
+			ex_act(2)
+		if(9 to INFINITY)
+			ex_act(1)
+	return 0
```

Comment:
Trailing return is likely unnecessary unless you call the proc elsewhere and look specifically for a "0" return.

Reply:
well singularity_act() actually is supposed to return the energy gained from consuming the atom so I figured it'd be nice to explicitely specify it here.